src/libcommonserver/io/iohelper.h

```diff
@@ -25,7 +25,33 @@ namespace KDC {
 
 struct FileStat;
 
+class DirectoryIterator {
```

Comment:
AS discussed, move this class in its dedicated file

Reply:
As rediscussed, for the moment `DirectoryIterator ` is implemented inside IoHelper.